.travis.yml

```diff
@@ -1,5 +1,47 @@
 language: dart
-sudo: false
-dart:
-  - dev
-script: ./tool/travis.sh
+
+# Gives more resources on Travis (8GB Ram, 2 CPUs).
+# Do not remove without verifying w/ Travis.
+sudo: required
```

Comment:
We should update `mono_repo` to handle "normal" repos – just to support generating this config.

I chatted w/ @natebosch about this yesterday
CC @alorenzen